ci/jenkins/Jenkins_steps.groovy

```diff
@@ -1389,6 +1393,238 @@ def docs_website() {
     }]
 }
 
+
+def compile_unix_lite() {
```

Comment:
How much time are these sstages going to add to the CI. Will they be run as part of every PR or nightly ?